### ISSUE TITLE
Espace % in references

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#295]
It would be usefull to escape '&#37;' chars in references since it's a comment sign in latex.
I had this problem when creating references to fortran derived variables such as 'foo&#37;bar'.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/295
- Originally reported by: [stefraynaud](https://bitbucket.org/stefraynaud)
- Originally created at: 2009-12-03T16:38:48.742

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-03 14:49:17+00:00_

Can you give me an example what markup causes this?

**Comment by shimizukawa** [comment]
_From Anonymous on 2010-01-05 16:13:20+00:00_

First, I declare a new description unit for fortran variables in a dedicated directive ::
{{{
#!python
def setup(app):

```
    app.add_description_unit('fvar', 'fvar', indextemplate='pair: %s; Fortran variable', )
```

}}}

Second, I declare a variable and reference it in my document :
{{{

.. fvar:: aa%bb

```
    ok
```

Reference:  :fvar:`aa%bb`
}}}

Html will be built and reference will work.
Latex will be built but not the pdf because the '%' is not escaped in hyperef name.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-01-06 10:50:10+00:00_

OK, thanks! This should now be fixed in <<cset 4c47d2d8e28bbd49a1ac5caf84af17c6e0c291da>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:32+00:00_

Removing version: 0.6.3 (automated comment)
